.github/workflows/6_bumper_repository.yml

```diff
@@ -0,0 +1,140 @@
+name: Repository bumper
+run-name: Bump ${{ github.ref_name }} (${{ inputs.id }})
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Target version (e.g. 1.2.3)'
+        default: ''
+        required: false
+        type: string
+      stage:
+        description: 'Version stage (e.g. alpha0)'
+        default: ''
+        required: false
+        type: string
+      date:
+        description: 'Release date in format YYYY-MM-DD (e.g., 2025-04-14)'
+        default: ''
```

Comment:
I think we have an error on this line

`error parsing called workflow ".github/workflows/6_builderpackage_server_pr.yml" -> "./.github/workflows/5_builderpackage_server.yml" : failed to fetch workflow: workflow was not found.`